src/CMakeLists.txt

```diff
@@ -335,7 +354,7 @@ endif()
 
 if(WIN32 AND X86_64)
   list(APPEND extra_libs "${fast_tokenezers_libs_dir}/icudt70.dll"
-                         "${fast_tokenezers_libs_dir}/icuuc70.dll")
+                         "${fast_tokenezers_libs_dir}/icuuc70${CMAKE_DEBUG_POSTFIX}.dll")
```

Comment:
1. It's not valid for prebuilt libraries
2. Why does not icudt70 have the same suffix?

Reply:
icudt70 is a datafile, not a real dll
For prebuild libs it works because we do not allow to build Debug tokenizers without rebuilding of fast_tokenizers

Reply:
if this case we don't need this postfix

Reply:
removed